core/trino-web-ui/src/main/resources/webapp-preview/src/api/webapp/api.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { api, ApiResponse } from "../base";
+
+export async function statsApi(): Promise<ApiResponse> {
+  return api.get('/ui/api/stats');
+}
```

Comment:
missing ending new line

Reply:
fixed by new eslint rule in 4dba8597a99dbfa95db7efdb8517a6391a379f2d.